site.yml

```diff
@@ -21,6 +21,34 @@
       delegate_to: localhost
       changed_when: false
 
+    - name: Create folder to store config
+      file: path={{ jenkins_job_config_file_src }} state=directory mode=0755 owner=jenkins group=jenkins
+
+    - name: Locally clone Jenkins job configs
+      git:
+        repo: "{{ jenkins_job_config_git_src }}"
+        dest: "/tmp/nfv_jobs_config"
+        version: HEAD
+        force: yes
+      delegate_to: localhost
+      changed_when: false
+
+    - name: Synchronize job config to remote server
+      synchronize: src="/tmp/nfv_jobs_config" dest="{{ jenkins_job_config_file_src }}" archive=yes
```

Comment:
Same as above, just modify this line to be relevant path to `./files/` and then expand the style format.